### ISSUE TITLE
OpenId connect supported in cognito Identity providers

### ISSUE THREAD

**Issue by sunsoori** [aws/amazon-cognito-auth-js#132]
All of a sudden I'm seeing "openId connect" option in the Identity providers of Cognito. Only since a fews hours back. Is this a new feature launched just now? I'm planning to integrate atlassian crowd with cognito. Hope it works now.

**Comment by rachitdhall** [comment]
Yes, we launched the support for federating through OpenId Connect providers today, you can find more details [here](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-oidc-idp.html).

**Comment by sunsoori** [comment]
I have a standalone Atlassian Crowd installed inside our company. Also the CrowdID which an openID connect type. Is it possible to integrate that with cognito now?

**Comment by AMiggel** [comment]
Please, help me, i need run this app and i don't know how to do it.
